project.godot

```diff
@@ -40,6 +40,7 @@ ToolTipManager="*res://src/gui_common/tooltip/ToolTipManager.tscn"
 TransitionManager="*res://src/gui_common/TransitionManager.cs"
 CursorLoader="*res://src/engine/CursorLoader.cs"
 ScreenShotTaker="*res://src/engine/ScreenShotTaker.cs"
+QuickLoadHandler="*res://src/gui_common/QuickLoadHandler.cs"
```

Comment:
This is pretty strange. I wonder where this change is coming from?

Reply:
Also it seems the game doesn't currently start. I'm all out of energy for today so I'm going to check out things tomorrow (if someone hasn't fixed things by then).

Reply:
I've removed it and fixed a few external assignments and now it is working